client/e2e/seasonsPage.spec.ts

```diff
@@ -0,0 +1,33 @@
+import {test , expect} from '@playwright/test';
+import {SeasonsPage} from './pages/seasonsPage';
+
+test('Homepage->Seasons',async({page}) => {
+    const seasonsPage = new SeasonsPage(page);
+    await seasonsPage.goto();
+  });
+
+test('addNewSeasonWithEvent',async({page})=>{
+    const seasonsPage = new SeasonsPage(page);
+    await seasonsPage.goto();
+    await seasonsPage.addNewSeason();
+    await seasonsPage.addEventToSeason();
+});
```

Comment:
As a rule of thumb, tests should be atomic - that is, don't rely on any pre-existing state at the start of your test, and clean up anything you add at the end.  I see there's some cleanup in future tests from the event added in here, but ultimately each test should contain its own cleanup.

A pattern we've found that's working so far is to put the test inside of a try block, and then putting all the cleanup inside of a finally block.  That way even if the test fails, the database is still cleaned up.  In this case, if possible I would suggest separating the logic that adds the seasons from the expects that are checking things, do all your add logic (both event and season adds) before the try block, all your expects inside of the try block, and then cleanup in the finally.  That said, this pattern won't necessarily work in all cases (say, you need to do some checks between adding different things).

Reply:
Made the changes suggested and I believe I have covered all the points you brought up. Let me know if this is more what you had in mind!